pkg/oc/bootstrap/docker/openshift/helper.go

```diff
@@ -378,6 +380,18 @@ func (h *Helper) Start(opt *StartOptions, out io.Writer) (string, error) {
 		startCmd = append(startCmd, fmt.Sprintf("--loglevel=%d", opt.LogLevel))
 	}
 
+	if extendedArgs := os.Getenv(openshiftMasterExtendedArgsVar); len(extendedArgs) > 0 {
+		args, err := argv.Argv([]rune(extendedArgs), map[string]string{}, argv.Run)
```

Comment:
FYI I used `github.com/cosiner/argv` because a) i'm lazy to write custom parser for args, b) lazy to make kubernetes extended arguments parse wired here, c) cosiner/argv have strong test coverage for edge cases.

Reply:
We have a parser for arguments, but I'm super not happy about this.

Reply:
shell_parser.go

Reply:
@smarterclayton any other suggestion about a better way how to pass extra args to the master is welcome :-) I can check the shell_parser.

Reply:
hrm. shell_parser.go is imagebuilder dep?

./vendor/github.com/openshift/imagebuilder/shell_parser.go
./pkg/build/builder/vendor/github.com/openshift/imagebuilder/shell_parser.go

Not sure if I want to import it here as that will add docker libs and other goodies?